tldw_Server_API/app/api/v1/endpoints/workspaces.py

```
@@ -109,6 +132,120 @@ def _require_workspace(db: CharactersRAGDB, workspace_id: str) -> dict:
+    }
+    try:
+        jm.create_job(
+            domain=WORKSPACE_SOURCE_JOB_DOMAIN,
+            queue=WORKSPACE_SOURCE_JOB_QUEUE,
+            job_type=WORKSPACE_SOURCE_JOB_TYPE,
+            payload=payload,
+            owner_user_id=str(current_user.id),
+            idempotency_key=f"workspace-source:{workspace_id}:{source_id}:{media_id}",
+            max_retries=3,
+        )
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)